### ISSUE TITLE
[MINOR] Trays cant load food items into microwave/grinder/food processor

### ISSUE THREAD

**Issue by misterreporter** [Skyrat-SS13/Skyrat-tg#1635]
## Reproduction:

Reproduction: Load up a tray with bacon, meat, a crop, really any food that be loaded onto the tray then unloaded into a microwave, grinder, or food processor.  The tray wont work and it wont unload the food so for instance: 

You cant load cutlets into a food processor to make bacon. 

You cant load bacon into the microwave to cook it 

You cant load wheat into the food grinder to quickly grind wheat.

Basically, the tray doesnt do anything you click on the object with loaded food items (microwave, processor, grinder, etc.) and it does nothing.

**Comment by Owai-Seek** [comment]
Sounds like another issue from upstream because they refactored food. I might have a look at it in the next few days.

**Comment by misterreporter** [comment]
> Sounds like another issue from upstream because they refactored food. I might have a look at it in the next few days.

Thank you.  For people like me who love to cook it makes it so burdensome I tend not to bother now.  Manually loading one item at a time is exhausting.  Appreciate it!

**Comment by Owai-Seek** [comment]
Yep, upstream issue. The person who's been handling the food rework might look into it, but if not, I'll take a crack at it.
https://github.com/tgstation/tgstation/issues/54812
https://github.com/tgstation/tgstation/issues/54818

**Comment by Azarak** [comment]
fixed upstream
